lib/src/visitor/perform.dart

```diff
@@ -555,11 +566,35 @@ class _PerformVisitor
     });
   }
 
+  /// If the scheme is `package`, converts the [import] url into a dart package uri.
+  /// If the scheme is not `package`, returns the the same [import] url.
+  /// If the [_packageResolver] is null, throws a [SassRuntimeException].
+  /// If the [_packageResolver] cannot find the file, throws a [SassRuntimeException].
+  Uri _resolvePackageUri(DynamicImport import) {
+    var packageUri = import.url;
+    if (packageUri.scheme == 'package') {
+      if (_packageResolver == null) {
+        throw _exception(
+            'Can\'t resolve: "$packageUri", packageResolver is not supported by node vm.'
+            ' If you are using dart-vm please provide a `SyncPackageResolver` to the `render` function',
+            import.span);
+      }
+
+      var resolvedPackageUri = _packageResolver.resolveUri(packageUri);
+      if (resolvedPackageUri == null ||
+          !dirExists(p.dirname(p.fromUri(resolvedPackageUri)))) {
```

Comment:
I'm not sure I understand the point of this check. If the package URI is resolvable, we'll already get an error when we try to load the file, right?